week_six/quiz_class.dart

```diff
@@ -14,7 +14,12 @@ void main() {
 }
 
 List<int> findPair(List<int> numbers, int target) {
-  // Your code here
-
-  return [2, 5];
+  for(int check1=0;check1<numbers.length;check1++){
+    for(int check2=check1+1;check2<numbers.length;check2++){
+      if(numbers[check1]+numbers[check2] ==target){
+        return [numbers[check1],numbers[check2]];
+      }
+    }
+  }
+  return [2,5];
```

Comment:
Remove the default value

Reply:
Got it sir, I didn’t changed it because you said to type on the given space.  I will change it rightaway.